doc/neps/nep-0026-missing-data-summary.rst

```diff
@@ -717,6 +717,16 @@ implementation strategy for NA dtypes.
 A further discussion overview page can be found at:
 https://github.com/njsmith/numpy/wiki/NA-discussion-status
 
+Additional References (2020):
+--------------------
+
+In 2018 Wes McKinney demonstrates, that sentinel values are actually less performant than bitmasks (contrary to popular belief also expressed here):
+https://wesmckinney.com/blog/bitmaps-vs-sentinel-values/
+
+In a follow-up Pasha Stetsenko optimizes the code for sentinel values by Wes McKinney, allowing them to perform similarly (but not better):
+https://github.com/st-pasha/microbench-nas/blob/master/README.md
```

Comment:
This seems more like discussion than a reference, and agree that it sort of undermines the statement above the first reference and is thus confusing.